lib/pages/home/create_movie/widgets/create_movie_button.dart

```diff
@@ -62,8 +62,8 @@ class _CreateMovieButtonState extends State<CreateMovieButton> {
 
         await executeFFmpeg(
                 '-f concat -safe 0 -i $txtPath -map 0 -c copy $outputPath')
-            .then((result) {
-          if (result == 0) {
+            .then((session) {
+          if (session.getReturnCode() == 0) {
```

Comment:
This gives `Code error: Instance of Future<String?>` and nothing else relevant on terminal logs. Maybe we should check like the other video edit, with `ReturnCode.isSuccess(returnCode)`. I tested with 2 videos, on the emulator, one with front cam and the other with back one.